pubspec.yaml

```diff
@@ -1,15 +1,15 @@
-name: aws_cloudwatch
-description: An easy, lightweight, and convenient way to reliably send logs to AWS CloudWatch.
-version: 0.4.6+2
-homepage: https://github.com/Zsmerritt/Flutter_AWS_CloudWatch
-
-environment:
-  sdk: '>=2.12.0 <3.0.0'
-
-dependencies:
-  aws_request: ^0.2.1 # Make request to AWS
-  synchronized: ^3.0.0 # Synchronize requests to ensure valid keys
-  universal_io: ^2.0.0 # Web support\
-
-dev_dependencies:
+name: aws_cloudwatch
+description: An easy, lightweight, and convenient way to reliably send logs to AWS CloudWatch.
+version: 0.4.6+2
+homepage: https://github.com/Zsmerritt/Flutter_AWS_CloudWatch
+
+environment:
+  sdk: '>=2.12.0 <3.0.0'
+
+dependencies:
+  aws_request: ^0.3.0 # Make request to AWS
+  synchronized: ^3.0.0 # Synchronize requests to ensure valid keys
+  http: ^0.13.4 #Http package for parsing AWS responses correctly on Multiple Platforms 
```

Comment:
This should probably just be 0.13.0. There is no reason that 0.13.0 shouldn't work and putting the version higher could cause some people compatibility issues with no real benefit

Reply:
that's true, I will change the version here.